template.rb

```diff
@@ -59,7 +59,12 @@ def apply_template!(template_root)
   # Add-ons - [Default]
   apply '.template/addons/docker/template.rb'
   apply '.template/addons/semaphore/template.rb'
-
+  
+  # Add-ons - [Optional]
+  if yes?("Would you like to add PhraseApp configuration?")
```

Comment:
Can we make it a one-liner and add a `(yes/no)` to make it more user friendly?

`apply '.template/addons/phrase_app/template.rb' if yes?("Would you like to add PhraseApp configuration?(yes/no)")`

Reply:
Also, should we move it before `after_bundle` so we won't have to use after_bundle in [.template/addons/phrase_app/template.rb](https://github.com/nimblehq/rails-templates/pull/231/files#diff-18847e9c4ee2991b23f88d42738a1738). What do you think 🤔

Reply:
I'm just wondering that if we make it in 1 line, the line could be really long.

Reply:
If it's really long we should keep it as it is 👍

Reply:
Refactor that in https://github.com/nimblehq/rails-templates/pull/231/commits/14e04aa961b618654cb3636bbad31be030676dd9 that will solve another issue which is the message template look like the same between each addon, right now each PRs has a different message structure